src/components/accordion/accordion.sandbox.html

```diff
@@ -0,0 +1,26 @@
+<div>
+    <m-accordion icon-position="right">
+        <div slot="header">
```

Comment:
Moi je mettrais un titre genre "toggle" avant l'accordion pour dire que tu test les toggles.. pis au lieu de mettre test du href, test du X, test du Y je mettrais "link should toggle".  Question de se retrouver dans le test